plugins/kuttl.yaml

```diff
@@ -18,27 +18,39 @@ spec:
       matchLabels:
         os: "linux"
         arch: "amd64"
-    uri: https://github.com/kudobuilder/kuttl/releases/download/v0.1.0/kuttl_0.1.0_linux_x86_64.tar.gz
-    sha256: "4043a3052a0f5b6c66b80fb1ad8dcfba10cb1e3a7f40812f991e1dd304a5af90"
+    uri: https://github.com/kudobuilder/kuttl/releases/download/v0.2.0/kuttl_0.2.0_linux_x86_64.tar.gz
+    sha256: "1b1c4f5be141c1469b445ad9ab9bf2e26261a37ce1af1b274839bec7bae8d306"
     bin: "./kubectl-kuttl"
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this is redundant, can you delete?

Reply:
ah... we ran into this last time and fixed on the pr... but not in my script the gens this file... on it.. thx!